libs/net/dal/Extensions/ContentExtensions.cs

```diff
@@ -98,6 +99,24 @@ public static TNOContext UpdateContext(this TNOContext context, Content original
             }
         });
 
+        // oquotes.Except(updated.Quotes).ForEach(a =>
```

Comment:
No we don't. Reason I don't delete them is because then we would just end up re-adding them if the extract-quotes service ran against the content again.  This way the editor can mark as quote as `isRelevant = false` and those quotes wont show up in the Subscriber UI, and they will already exist when the extract-quotes service runs.